pvlib/__init__.py

```diff
@@ -1,3 +1,10 @@
+# First ensure proper Python version.
```

Comment:
No need for this.

Reply:
I think it would help people who need to upgrade, but I'm fine removing.